src/libsystemd-network/sd-dhcp6-client.c

```diff
@@ -498,6 +498,14 @@ int sd_dhcp6_client_set_rapid_commit(sd_dhcp6_client *client, int enable) {
         return 0;
 }
 
+int sd_dhcp6_client_set_send_release(sd_dhcp6_client *client, bool enable) {
```

Comment:
As you already noticed, please use `int enable` even if the internal variable `client->send_release` is `bool`.